pkg/pod/limitrange/doc.go

```diff
@@ -0,0 +1,17 @@
+/*
+Copyright 2020 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+// Package limitrange defines logic for supporting Kubernetes LimitRange for the specific Tekton use cases
+package limitrange
```

Comment:
nice, i like the new package :D

random question and maybe off topic: would it make sense to put this in `internal/`? I'm wondering if we want to start putting as much code as possible in `internal/`, i.e. being careful about what code other projects might start depending on?

Reply:
I am a fervent defenser of `internal` package, so yes, I think we should put in under internal to start with and we can move it outside at some point if need be.

Reply:
Only "open question" is `pkg/internal/limitrange` or `internal/pkg/limitrange` ? 
We don't need to get access to this package in the e2e tests or `cmd` packages, so I'd rather have `pkg/internal` but 😛

Reply:
either is fine by me! :D 💃  `pkg/internal` sgtm - thanks @vdemeester !